pylint/checkers/base/basic_error_checker.py

```diff
@@ -78,6 +78,38 @@ def _has_abstract_methods(node: nodes.ClassDef) -> bool:
     return len(utils.unimplemented_abstract_methods(node)) > 0
 
 
+def _new_correctly_implemented(node: nodes.ClassDef) -> bool:
```

Comment:
I don't think we should call this "correctly". What about: `new_instantiates_super`?

Reply:
lovely, naming is hard :)